api/nakadi-event-bus-api.yaml

```diff
@@ -954,6 +976,107 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  /settings/flooders:
```

Comment:
Maybe we can add description like:
`The oauth resource owner username has to be equal to 'nakadi.oauth2.adminClientId' property to be able to access this endpoint`